### PR TITLE
Use es6 features

### PR DESCRIPTION
**Pull request by joeyciechanowicz**
Says what it does on the tin!

### CODE REVIEW

File: task/pa11y.js [hunk 1]
```diff
@@ -16,59 +16,63 @@
...
 const chalk = require('chalk');
-const CronJob = require('cron').CronJob;
+const {CronJob} = require('cron');
 
 module.exports = initTask;
 exports.runPa11yOnTasks = runPa11yOnTasks;
 
 // Initialise the task
 function initTask(config, app) {
+
```
Comment (josebolos) [inline task/pa11y.js]: I don't know if the linter enforces this, but we don't usually add an empty line after an open bracket?

File: task/pa11y.js [hunk 2]
```diff
@@ -16,59 +16,63 @@
...
 		config.cron = '0 30 0 * * *'; // 00:30 daily
 	}
 	const job = new CronJob(config.cron, run.bind(null, app));
 	job.start();
+
 }
 
 // Run the task
 function run(app) {
+
```
Comment (josebolos) [inline task/pa11y.js]: Same as above, empty line after an open bracket?

File: task/pa11y.js [hunk 3]
```diff
@@ -16,59 +16,63 @@
...
-		function(tasks, next) {
-			runPa11yOnTasks(tasks, app, next);
-		}
+		(tasks, next) => runPa11yOnTasks(tasks, app, next)
+
+	], error => {
 
-	], function(error) {
 		if (error) {
 			console.error(chalk.red('Failed to run tasks: %s'), error.message);
```
Comment (josebolos) [inline task/pa11y.js]: Do we want to use template literals here?

File: task/pa11y.js [hunk 4]
```diff
@@ -16,59 +16,63 @@
...
 // Run Pa11y on an array of tasks
 function runPa11yOnTasks(tasks, app, done) {
 
-	const queue = async.queue(function(task, nextInQueue) {
+	const queue = async.queue((task, nextInQueue) => {
 		console.log('Starting task %s', task.id);
-		app.model.task.runById(task.id, function(error) {
+		app.model.task.runById(task.id, error => {
 			if (error) {
 				console.log(chalk.red('Failed to finish task %s: %s'), task.id, error.message);
```
Comment (josebolos) [inline task/pa11y.js]: Ditto about template literals?

File: test/integration/setup.js [hunk 5]
```diff
@@ -22,31 +22,28 @@ const request = require('request');
...
 // Run before all tests
 before(function(done) {
-	const that = this;
-
-	this.baseUrl = 'http://localhost:' + config.port + '/';
+	this.baseUrl = `http://localhost:${config.port}/`;
 	this.app = null;
 	this.last = {};
-	this.navigate = createNavigator(that.baseUrl, that.last);
+	this.navigate = createNavigator(this.baseUrl, this.last);
```
Comment (josebolos) [inline test/integration/setup.js]: Why are we removing `that` here? Is this a previous bug that we're fixing?

File: test/integration/setup.js [hunk 6]
```diff
@@ -22,31 +22,28 @@ const request = require('request');
...
 
-	assertTestAppIsRunning(this.baseUrl, function() {
+	assertTestAppIsRunning(this.baseUrl, () => {
 		config.dbOnly = true;
-		app(config, function(error, initialisedApp) {
-			that.app = initialisedApp;
+		app(config, (error, initialisedApp) => {
+			this.app = initialisedApp;
```
Comment (josebolos) [inline test/integration/setup.js]: Same, why is `that` gone?

File: test/integration/setup.js (hunk 5 above)
Comment (joeyciechanowicz) [reply test/integration/setup.js]: It was missed previously when switching from `that → this`

File: test/integration/setup.js (hunk 6 above)
Comment (joeyciechanowicz) [reply test/integration/setup.js]: Fat arrow functions keep the same block scope as where they are declared. So `this` still referes to the parent functions `this`. 
`function` on the other hand do not keep the same scope, which was why the `const that = this` was required.
As we're using fat-arrows now, theres no need for the `that`.

**Pull request merged**